src/safeposix/syscalls/fs_calls.rs

```diff
@@ -206,20 +206,17 @@ impl Cage {
             (None, Some(pardirinode)) => {
                 let filename = truepath.file_name().unwrap().to_str().unwrap().to_string(); //for now we assume this is sane, but maybe this should be checked later
 
-                let effective_mode = S_IFREG as u32 | mode;
-
-                //assert sane mode bits
+                //assert same mode bits
```

Comment:
Sane was not a typo here, we're asserting that the mode bits make sense

Reply:
Oopsie -- fixing that now